Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.4-alpine
+LABEL maintainer="volodymyr_dolgyi@ukr.net"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media

```

Comment:
Consider moving the `RUN mkdir -p /files/media` command before the `COPY . .` command to ensure the directory exists before any files are copied.